python/tritonserver/__init__.py

```diff
@@ -23,3 +23,44 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""In process Python API for Triton Inference Server."""
+
+import tritonserver._c as triton_bindings
+from tritonserver._api.wrapper import (
+    InferenceRequest,
+    InstanceGroupKind,
+    LogFormat,
+    MetricFamily,
+    MetricFormat,
+    MetricKind,
+    Model,
+    ModelBatchFlag,
+    ModelControlMode,
+    ModelIndexFlag,
+    ModelTxnPropertyFlag,
+    Options,
+    RateLimitMode,
+    Server,
+)
+
+exceptions = [
+    triton_bindings.TritonError,
+    triton_bindings.NotFoundError,
+    triton_bindings.UnknownError,
+    triton_bindings.InternalError,
+    triton_bindings.InvalidArgumentError,
+    triton_bindings.UnavailableError,
+    triton_bindings.UnsupportedError,
+    triton_bindings.AlreadyExistsError,
+]
+
+
+# Rename module for exceptions to simplify stack trace
+for exception in exceptions:
+    exception.__module__ = "tritonserver"
```

Comment:
question: what does the exception look like without this change?

Reply:
With the change:
`tritonserver.InternalError: failed to load all models`

Without the change:
`tritonserver._c.triton_bindings.InternalError: failed to load all models`

Small difference -but felt that since the errors are part of the API - that having them aliased made sense ... open to discuss though.